src/View/mainpage/DrawROIWindow/Drawing.py

```diff
@@ -0,0 +1,291 @@
+import math
```

Comment:
Many lines in this file far too long for PEP 8 compliance:
14-18, 22-23, 26, 47, 60, 66-69, 77, 85-86, 90-91, 107, 125, 133, 136, 141, 143, 146, 160, 163, 170, 172, 178, 185-186, 191-192, 196, 214, 219, 225, 233, 238, 266-267, 270, 272, 280, 282, & 284.

Also, all functions either don't have docstrings or have incorrect docstrings.